src/implementation/strided.jl

```diff
@@ -228,7 +231,7 @@ function blas_contract!(C, A, pA, B, pB, pAB, α, β, allocator)
         C_ = SV(tensoralloc_add(TC, C, ipAB, false, Val(true), allocator))
         _unsafe_blas_contract!(C_, A_, pA, B_, pB, trivialpermutation(ipAB),
                                one(TC), zero(TC))
-        stridedtensoradd!(C, C_, pAB, α, β, StridedNative(), allocator)
+        stridedtensoradd!(C, C_, pAB, α, β, backend, allocator)
```

Comment:
should this be `stridedtensoradd!` or just `tensoradd!`?

Reply:
right, if this is to work generally, it should be `tensoradd!`